src/app/containers/Blocks/index.test.jsx

```diff
@@ -3,55 +3,81 @@ import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';
 import Blocks from './index';
 import { blockContainingText, singleTextBlock } from '../../models/blocks';
 
-describe('Blocks', () => {
-  const blocks = [
-    blockContainingText('headline', 'This is a headline!'),
-    blockContainingText('subheadline', 'This is a subheadline!'),
-    singleTextBlock('This is some text content!'),
-    {
-      type: 'test',
-      model: {
-        blocks: [
-          {
-            model: {
-              text: 'This is some test content!',
-            },
-          },
-        ],
+const unknownTestBlock = {
+  type: 'test',
+  model: {
+    blocks: [
+      {
+        model: {
+          text: 'This is some test content!',
+        },
       },
-    },
-    {
-      type: 'image',
-      model: {
-        blocks: [
-          {
-            type: 'rawImage',
-            model: {
-              width: 640,
-              height: 420,
-              locator: 'E7DB/production/_101655395_paulineclayton.jpg',
-              originCode: 'cpsprodpb',
-              copyrightHolder: 'BBC',
-            },
-          },
-          blockContainingText('altText', 'Pauline Clayton'),
-          blockContainingText(
-            'caption',
-            'Former embroider Pauline Clayton described the gift as "lovely"',
-          ),
-        ],
+    ],
+  },
+};
+
+const unknownNestedBlock = {
```

Comment:
It would be clearer to rename this, since this isn't an `unknown` nested block - all of the blocks here are known. It is just that they are not within the `componentsToRender` for this case. 

```suggestion
const parentBlockNotAllowed = {
```

Reply:
done :)